pkg/cmd/roachtest/roachtestutil/mixedversion/steps.go

```diff
@@ -715,3 +715,51 @@ func startStopOpts(opts ...option.StartStopOption) []option.StartStopOption {
 		option.NoBackupSchedule,
 	}, opts...)
 }
+
+type panicNodeStep struct {
```

Comment:
This mutator only affects the system tenant. It's fine for the first iteration, but I don't see any blockers to get this working for secondary tenants. Lets leave a `TODO(kyleli):` to come back and support panicking secondary tenants.